portfolio/src/main/webapp/hiking.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
```

Comment:
Challenge (if you have time):  A lot of these pages are essentially the same template but with a few elements changed.  You could use JS to dynamically show or hide elements, or change text/content, instead of navigating to different pages, and that would dramatically shrink your code and avoid code duplication.

Key principle in general:  DRY -- don't repeat yourself.  If you find yourself copy+pasting code, think about how you could reorganize so that code is only written in one single place.

Reply:
I was thinking about this last night after work and how to do it. Have been doing it with the small things and will probably have to come back later for the bigger ones.